OptionSelector/DetailTableViewController.m

```diff
@@ -0,0 +1,67 @@
+//
+//  SubTableViewController.m
+//  OptionSelector
+//
+//  Created by Brian Blanco on 8/14/15.
+//  Copyright © 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "DetailTableViewController.h"
+
+@interface DetailTableViewController ()
+
+@end
+
+@implementation DetailTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
```

Comment:
Style: The [default return value of this method is 1](https://developer.apple.com/library/ios/documentation/UIKit/Reference/UITableViewDataSource_Protocol/#//apple_ref/occ/intfm/UITableViewDataSource/numberOfSectionsInTableView:), so you can also remove this method definition if you want.